packages/geo/src/lib/import-export/import-export/import-export.component.ts

```diff
@@ -550,6 +549,23 @@ export class ImportExportComponent implements OnDestroy, OnInit {
         geomTypes = [{ geometryType: '', features: olFeatures }];
       }
 
+      if (data.circlesToPolygons) {
+        geomTypes = [];
```

Comment:
this action could be done a the draw-component level. Always store a polygon.

refer to ol/geometry/polygon/fromCircle